data/chamberezigbo.json

```diff
@@ -1,59 +1,79 @@
 {
   "name": "Chamberlain Ezigbo",
   "type": "personal",
+<<<<<<< HEAD:public/data/chamberezigbo.json
   "bio": "JavaScript Engineer | Front-end Developer",
-
+=======
+  "bio": "JavaScript Engineer | Full-stack Developer",
+>>>>>>> 444abc75fa64b328ee08acf24eca5a99d01d3a77:data/chamberezigbo.json
+  "avatar": "https://github.com/chamberezigbo.png",
```

Comment:
Bio is added twice, one should be removed. Also, we no longer need the `avatar` key. 

```suggestion
```